backend/tests/Ragas/utils/modules/ragas_evaluation.py

```diff
@@ -198,6 +204,31 @@ async def evaluate_with_ragas(
                     avg_data[metric] = None  # None is fine for a dict with explicit type annotation
                     print(f"No valid values for {metric}")
 
+        # Calculate aggregate usage data if available
+        if "llm_usage" in results_df.columns:
+            usage_records = results_df["llm_usage"].dropna()
```

Comment:
I'm guessing `dropna()` get's rid of the "N/A" strings, but what does it actually do?

Reply:
It drops all the null rows, although I'm fairly sure there wont be any by this point